src/features/code_actions.zig

```diff
@@ -109,7 +109,35 @@ fn handleUnusedFunctionParameter(builder: *Builder, actions: *std.ArrayListUnman
 
     const block = node_datas[payload.func].rhs;
 
-    const new_text = try createDiscardText(builder, identifier_name, token_starts[node_tokens[payload.func]], true);
+    // If we are on the "last parameter" that requires a discard, then we need to append a newline,
+    // as well as any relevant indentations, such that the next line is indented to the same column.
+    // To do this, you may have a function like:
+    // fn(a: i32, b: i32, c: i32) void { _ = a; _ = b; _ = c; }
+    // or
+    // fn(
+    //     a: i32,
+    //     b: i32,
+    //     c: i32,
+    // ) void { ... }
+    // We have to be able to detect both cases.
+    var param_end = offsets.tokenToLoc(tree, ast.paramLastToken(tree, payload.get(tree).?)).end;
+    var found_comma: bool = false;
+    while (param_end < tree.source.len) : (param_end += 1) {
+        switch (tree.source[param_end]) {
+            ' ', '\n' => continue,
+            ',' => if (!found_comma) {
+                found_comma = true;
+                continue;
+            } else {
+                param_end += 1;
+                break;
+            },
+            ')' => break,
+            else => break,
+        }
+    }
```

Comment:
Looks like you copied this code from the `getParamRemovalRange` function. You do not use `param_end` after iterating so there is no need to continue after encountering a comma.

This means that we can write the following:
```zig
var found_comma: bool = false;
while (param_end < tree.source.len) : (param_end += 1) {
    switch (tree.source[param_end]) {
        ' ', '\n' => continue,
        ',' => {
            found_comma = true;
            break;
        },
        ')' => break,
        else => break,
    }
}
```

If we use some of the functions from the `std.mem` we can also do this:
```zig
const found_comma = std.mem.startsWith(u8, std.mem.trimLeft(u8, tree.source[param_end..], " \n"), ",");
```